lists/ir.csv

```diff
@@ -1,9 +1,6 @@
 url,category_code,category_description,date_added,source,notes
-http://213.4.130.210,NEWS,News Media,2014-04-15,citizenlab,
 http://4shared.com,FILE,File-sharing,2015-03-20,OONI,
-http://aadaamaak.blogfa.com,LGBT,LGBT,2014-04-15,citizenlab,

```

Comment:
OONI measurements show that this site although dead is still actually blocked in Iran: https://explorer.ooni.torproject.org/measurement/4F3l2MvLez7ANO9BpmjMHu4c59DRur0l9qr6vCfAMJuhi5wVzNHXIsmUpO1HPNN9?input=http:%2F%2Faadaamaak.blogfa.com